### ISSUE TITLE
[ERROR] Signature check for DIGESTS failed.

### ISSUE THREAD

**Issue by 5HT** [AntonioMeireles/coreos-xhyve#124]
```
bash-3.2$ corectl run
[WARN] unable to run "defaults read /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist" Shared_Net_Address ...
[WARN] ... assuming macOS default value (192.168.64.1)
[WARN] unable to run "defaults read /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist" Shared_Net_Mask ...
[WARN] ... assuming macOS default value (255.255.255.0)
---> downloading and verifying alpha/1618.0.0
[ERROR] Signature check for DIGESTS failed.
```

```
$ corectl version

Server:
 Version:	0.7.18
 Go Version:	go1.9
 Built:		Sat Sep 23 17:16:47 BST 2017
 OS/Arch:	darwin/amd64

Client:
 Version:	0.7.18
 Go Version:	go1.9
 Built:		Sat Sep 23 17:16:47 BST 2017
 OS/Arch:	darwin/amd64
```

**Comment by jacobat** [comment]
Dupe of #121 ?

**Comment by 5HT** [comment]
Probably, I've got my version of `corectl` through `brew` channel.
So 124 dup 121 dup 120 and 120 is closed but bug is present.

**Comment by soudaburger** [comment]
+1
Just ran into this issue today.  Downloaded everything through brew/cask and don't see any way to resolve it. 

It would be nice if you could allow users to bypass hash checking for scenarios just like these.

**Comment by jacobat** [comment]
You can run with the `-v` argument to pass a specific version. I've been using `corectl run -v 1520.0.0 ...` to circumvent this problem.

**Comment by aarvee11** [comment]
# Issue:
```
$ corectl run -v 1632.2.1
[WARN] /Users/aarvee/.coreos/images/alpha/1675.0.1/coreos_production_pxe.vmlinuz missing - alpha/1675.0.1 ignored
---> downloading and verifying alpha/1632.2.1
[ERROR] failed fetching http://alpha.release.core-os.net/amd64-usr/1632.2.1/coreos_production_pxe_image.cpio.gz.DIGESTS.asc: HTTP status: 403 Forbidden
```

# Debug:
Even when we specify the channel as stable, the URL being called is the alpha channel. Which means that default "latest" is from Stable Channel and the signature is being checked from alpha channel. 

In order to test further, I did a manual curl for the DIGESTS.asc file from stable channel and below is the output:

```
$ curl -i http://stable.release.core-os.net/amd64-usr/1632.2.1/coreos_production_pxe_image.cpio.gz.DIGESTS.asc
HTTP/1.1 200 OK
Date: Fri, 09 Feb 2018 10:52:33 GMT
Content-Type: text/plain
Content-Length: 2550
Connection: keep-alive
Set-Cookie: __cfduid=db29a8621f6753ee8691bd11601b60ca01518173552; expires=Sat, 09-Feb-19 10:52:32 GMT; path=/; domain=.core-os.net; HttpOnly
X-GUploader-UploadID: AEnB2UriYPYyDkqcBwPNH9Hw8J_J5j8kRaFODcrByHt8uS_5bfxQWepzYFiPmDQZhKO4ouJaOYXM5KQIA8-7AeF4buB5-2x849I3P7yaKFvL2eQRe6RUeIs
Expires: Fri, 09 Feb 2018 11:05:33 GMT
Cache-Control: public, max-age=3600
Last-Modified: Fri, 02 Feb 2018 06:40:49 GMT
ETag: "d2bab57a3a131509cd3208ae5afcad59"
x-goog-generation: 1517553649352807
x-goog-metageneration: 1
x-goog-stored-content-encoding: identity
x-goog-stored-content-length: 2550
x-goog-hash: crc32c=vf87oQ==
x-goog-hash: md5=0rq1ejoTFQnNMgiuWvytWQ==
x-goog-storage-class: STANDARD
Alt-Svc: hq=":443"; ma=2592000; quic=51303431; quic=51303339; quic=51303338; quic=51303337; quic=51303335,quic=":443"; ma=2592000; v="41,39,38,37,35"
CF-Cache-Status: HIT
Accept-Ranges: bytes
Server: cloudflare
CF-RAY: 3ea647a2258a897b-MAA

-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

# MD5 HASH
5daaa03ff235b7fd9e1259343332a663  coreos_production_pxe.vmlinuz
# SHA1 HASH
89cae94ed857f389cdb23f11016ce2df8480bad7  coreos_production_pxe.vmlinuz
# SHA512 HASH
7ff704ae3a6f55c18911b8d733050f54b11274c013f7c83a96d8c83558c5e7b62459938a1bb2e64deb1cd449dcd960c6e54f275222d87d9091f2621485e332f4  coreos_production_pxe.vmlinuz
# MD5 HASH
43ece3548bc13b8bc03335ca2abb3794  coreos_production_pxe_grub.efi
# SHA1 HASH
d0b9203fe9dce39c667991bc1668115c0d43d7c2  coreos_production_pxe_grub.efi
# SHA512 HASH
d502f104fad52b1a3f598a997c5d7a5d34bbb2b4f13042e092a778316449d295e43bf78f17e146ad2c43cc5358d4cd82b6547f59b0956c0b4a0e180c21cb2c3a  coreos_production_pxe_grub.efi
# MD5 HASH
29a7f843d9a7fbdc6feb497b18a0af3b  coreos_production_pxe_image.cpio.gz
# SHA1 HASH
e97e9beecb78d1dca37a18da80621a6faec52028  coreos_production_pxe_image.cpio.gz
# SHA512 HASH
0f62679d8ba2bb385db75b31521bbcdd7f2db20150bf2ebf6043061abac2f3b7a05bfb883d57c52c8f7908ffed0168a30df6f15a7308c2611a01184bd2a18eef  coreos_production_pxe_image.cpio.gz
# MD5 HASH
7f61191d8cf3a2ad75650071295bd0fc  coreos_production_pxe.sh
# SHA1 HASH
8321d8d3653dba02e192d3bbc5257405ba283954  coreos_production_pxe.sh
# SHA512 HASH
c4d596642b2ae6dacf19c39861ed4e2bec233add8df4c5082d4a0faf4e2ac0dcb549cc28828475844d0f65ef20cb7e93d484cd1802b0a352f4819270cf36e401  coreos_production_pxe.sh
# MD5 HASH
747e80e91eac86434517a5fd8554fcfd  coreos_production_pxe.README
# SHA1 HASH
7d6ab9c1ddddb856d01540208874f21ba82b5cb0  coreos_production_pxe.README
# SHA512 HASH
0e616edc781b203871968625b7ebe51cd0bdd2b992224cb505d61e241561d5d6168cc131f98566397df05536a55c4a44683e4187ccc0ac0cd1171ea826fc22b1  coreos_production_pxe.README
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEiCatlWn1da0/VkPn3i+Ph+9LTtkFAlpzgyIACgkQ3i+Ph+9L
TtlfPQ//VcQGSodbBXaQXY49iWY+UVefMVf+TQM36BYHHeG2509Ox31TZdKauK0W
7eMV+yDVsvXgeWzskg7N5ZUA7i2wy0Y4g/oz2ob3SdXvZDzzQz1T0Rv+kMaV+8fL
JMUNSacUNRK/72DzUiKNbMXGOfnb5vqnPNeHUinRIwfQR40NQB0j2D14MqMRwHfX
mJo+21jxJ4wN4arpXZMkQZgmEOAe5zARP9MkyqYVcdec+LZak2WZmaaNB+HMT0ys
jL9+B6aNHVYP9QP49x0U7M78PEuG/w0QOK+Zd0gjnycmPnzyA0B0+kAELWawIhEG
FnVg35xocuhZQVB049N4+nvd5CspnRFkulc9WgVuQqzYZoYJIsrLieCLlLv8ptnH
LSQV+nH/s/16ymZn3gWQ+wSxUYStNeOsnlMMLiQa6tk0aHjR/Gy6b6/loJmnbhfh
7qTml+Ck0SUqPjxzC6TufU7cZnHHMLIPX/l39d+fJjmy8v9rGWKe+6iLEmdmsSOj
CfrepUgie/7vuqva7z2DUYIcodKXxyrSLTUqreFc/8w/hvrbfZcgWoCOLncfNH+5
+QjJwsWTvBSa6rx4Ulr+l6KzlL5s0xMW+a9/DkDL9fv4SnU3AJ/hnoD8HnG89BTZ
k6jGO50Z1oV38oGSIViaLnTXrPgVSfJuuxJHKUm05bGgyP9vfBk=
=UFlZ
-----END PGP SIGNATURE-----


$ curl -i http://alpha.release.core-os.net/amd64-usr/1632.2.1/coreos_production_pxe_image.cpio.gz.DIGESTS.asc
HTTP/1.1 403 Forbidden
Date: Fri, 09 Feb 2018 10:52:43 GMT
Content-Type: application/xml; charset=UTF-8
Content-Length: 271
Connection: keep-alive
Set-Cookie: __cfduid=df55988c3ee773fb0ad2a9cf7f53e2c7d1518173561; expires=Sat, 09-Feb-19 10:52:41 GMT; path=/; domain=.core-os.net; HttpOnly
X-GUploader-UploadID: AEnB2UoxTwdPlLTLFDIYiWH5XAQCTTwlMhypgF6-qYrTCRHKc6Q44RtEPz-hXO03vYsOpW_qfA5Ani9P4uzC4CFP_awOaWDf_RhBaFaSKIQxfiNrhS1QXaI
Expires: Fri, 09 Feb 2018 10:52:42 GMT
Cache-Control: private, max-age=0
Alt-Svc: hq=":443"; ma=2592000; quic=51303431; quic=51303339; quic=51303338; quic=51303337; quic=51303335,quic=":443"; ma=2592000; v="41,39,38,37,35"
CF-Cache-Status: MISS
Server: cloudflare
CF-RAY: 3ea647da103d2fab-MAA

<?xml version='1.0' encoding='UTF-8'?><Error><Code>AccessDenied</Code><Message>Access denied.</Message><Details>Anonymous caller does not have storage.objects.get access to coreos-alpha/amd64-usr/1632.2.1/coreos_production_pxe_image.cpio.gz.DIGESTS.asc.</Details></Error>
```
**HOPE THIS HELPS THE DEV FIXING THE ISSUE !**

**Comment by communig8-public** [comment]
Looks like this was fixed in the source code in issue #120 but the Corectl.app has not been rebuilt yet. Any change that can get done? Its a real pain trying to get the build environment together on my MacBook.
